airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java

```diff
@@ -99,6 +103,47 @@ public AbstractJdbcSource(final String driverClass,
     this.sourceOperations = sourceOperations;
   }
 
+  @Override
+  public AutoCloseableIterator<AirbyteMessage> read(final JsonNode config,
```

Comment:
I don't think we should be overriding the `read()` method here. We can add a generic logDatabaseData() method to the parent AbstractDbSource class, which has a default impl to not do anything. Then, we can override it in AbstractJdbcSource

Reply:
Got it. Followed suggested pattern.